src/main/java/net/sf/jabref/importer/fetcher/INSPIREBibtexFilterReader.java

```diff
@@ -71,16 +71,16 @@ private String readpreLine() throws IOException {
         return l;
     }
 
-    private String fixBibkey(String in) {
-        if (in == null) {
+    private String fixBibkey(final String _in) {
+        if (_in == null) {
             return null;
         }
         //System.out.println(in);
-        if (in.matches("@Article\\{.*,")) {
+        if (_in.matches("@Article\\{.*,")) {
```

Comment:
As this is a regex, it is better to extract this is a pattern which is then used for the matching. Because then the pattern can be reused and must not be compiled every thing it is used.